react-refresher/my-react-app/src/App.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+import MyButton from './button';
+import './App.css';
+
+function App() {
+  const [counter, setCounter] = useState(0);
+
+  function handleClickSubtract() {
+    setCounter(counter - 1);
+    console.log('subtracted');
+  }
+
+  function handleClickAdd() {
+    setCounter(counter + 1);
+    console.log('added');
+  }
+
+  return (
+    <div className="App">
+      <MyButton text={'Down'} onClick={handleClickSubtract} />
+      <div>{counter}</div>
+      <MyButton text={'Up'} onClick={handleClickAdd} />
```

Comment:
`onClick` is a confusing prop name, because it's the same name we use to attach an event to an element. I would change this to something like `onCustomClick`